### PR TITLE
Releasing azure cosmos 4.19.1

### PR DESCRIPTION
**Pull request by simplynaveen20**
Releasing azure cosmos 4.19.1

### CODE REVIEW

**Review by moderakh — APPROVED** [review]

**Review by kushagraThapar — APPROVED** [review]
LGTM

File: sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md [hunk 1]
```diff
@@ -1,14 +1,8 @@
...
-#### Features Added
-
-#### Breaking Changes
-
-#### Bugs Fixed
-
-#### Other Changes
+### 3.12.0 (2021-09-24)
+#### New Features
+* Added support for UUID id type to spring data cosmos SDK.
```
Comment (kushagraThapar) [inline sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md:5]: Please mention the azure-cosmos version update.

Comment (kushagraThapar) [reply sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md:5]: Like this -> Updated `azure-cosmos` to version 4.19.1.

**Review by kushagraThapar — CHANGES_REQUESTED** [review]

File: sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md (hunk 1 above)
Comment (simplynaveen20) [reply sdk/cosmos/azure-spring-data-cosmos/CHANGELOG.md:5]: done

**Review by kushagraThapar — APPROVED** [review]

**Pull request merged**